pkg/console/operator/sync_v400.go

```diff
@@ -214,6 +205,25 @@ func (co *consoleOperator) SyncConsolePublicConfig(consoleURL string) (*corev1.C
 	return resourceapply.ApplyConfigMap(co.configMapClient, co.recorder, requiredConfigMap)
 }
 
+// call after SyncConsoleConfig()
+func (co *consoleOperator) handleConsoleURLMetric(consoleConfig *configv1.Console, consoleURL string) {
+	if len(consoleURL) == 0 {
+		return
+	}
+	// if status.consoleURL is empty, and the consoleURL has a value, set it
+	if consoleConfig.Status.ConsoleURL == "" && len(consoleURL) != 0 {
+		co.consoleMetrics.ConsoleURL.WithLabelValues(consoleURL).Set(1)
+		return
+	}
+	// if status.consoleURL is different than consoleURL,
+	// clear the old, then set new
+	if consoleConfig.Status.ConsoleURL != consoleURL {
```

Comment:
will this ever be `true`? since this check is already done in `SyncConsoleConfig()` and is setting the `consoleConfig.Status.ConsoleURL` to `consoleURL` ?